csharp/src/Ice/Ice1BinaryConnection.cs

```diff
@@ -197,7 +197,7 @@ internal Ice1BinaryConnection(ITransceiver transceiver, Endpoint endpoint, Objec
                     }
                     else
                     {
-                        throw new ConnectionClosedByPeerException();
+                        throw new ConnectionClosedByPeerException("");
```

Comment:
I think we should always include a non-empty error message when throwing an exception.